CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -2,6 +2,21 @@
 
 *Some deprecated APIs have been removed in Antrea v2.0. Before upgrading, please read these [guidelines](https://github.com/antrea-io/antrea/blob/release-2.0/docs/versioning.md#upgrading-from-antrea-v1-to-antrea-v2) carefully.*
 
+## 2.0.1 - 2024-06-14
+
+### Changed
+
+- Update CNI binaries version to v1.4.1. ([#6334](https://github.com/antrea-io/antrea/pull/6334), [@antoninbas])
+
+### Fixed
+
+- Remove unexpected `altname` after rename interface to avoid failure when moving host interface to OVS bridge. ([#6321](https://github.com/antrea-io/antrea/pull/6321), [@gran-vmv])
+- Fix a bug that AntreaProxy not deleting stale UDP conntrack entries for the virtual NodePort DNAT IP. ([#6379](https://github.com/antrea-io/antrea/pull/6379), [@hongliangl])
+- Improve stale UDP conntrack entries deletion accuracy in zone 0 for AntreaProxy. ([#6193](https://github.com/antrea-io/antrea/pull/6193), [@hongliangl])
+- Fix antrea-agent crashing issue with proxyAll enabled in networkPolicyOnly mode. ([#6259](https://github.com/antrea-io/antrea/pull/6259), [@hongliangl])
+- Avoid generating defunct process when starting Suricata for Antrea with IDS. ([#6366](https://github.com/antrea-io/antrea/pull/6366), [@hongliangl])
```

Comment:
we should use the same changelog entry as for https://github.com/antrea-io/antrea/pull/6438

Reply:
yeah, thanks for reminder. I haven't addressed this yet, planning to address all comments and push after all e2e tests are passed.